metagpt/provider/ark_api.py

```diff
@@ -16,11 +37,45 @@ class ArkLLM(OpenAILLM):
     见：https://www.volcengine.com/docs/82379/1263482
     """
 
+    aclient: Optional[AsyncArk] = None
```

Comment:
Seems ark has a openai compatible interface, should we use the official sdk?

Reply:
ok, to support also ak/sk

Reply:
The openai compatible interface is not working, so it's replaced by sdk.